app/views/lectures/show.html.erb

```diff
@@ -44,19 +54,27 @@
   <div class="tab-pane fade" id="feedback" role="tabpanel" aria-labelledby="feedback-tab">
 
     <% if @current_user.is_student %>
-
-      <%= form_with(model: [ @course, @lecture, @lecture.feedbacks.build ]) do |form| %>
-        <p>
-          <%= form.label :content %><br>
-          <%= form.text_field :content %>
-        </p>
-        <p>
-          <%= form.submit %>
-        </p>
+      <%= form_with(model: [ @course, @lecture, @lecture.feedbacks.build ], :html => { :onsubmit => "handleSubmit()" }) do |form| %>
+        <div id="feedback-notification" style="display:none" class="alert alert-success" role="alert"> Feedback submitted</div>
+        <% if @lecture.feedbacks.where(user_id: current_user.id).count >= 1 %>
```

Comment:
There is hopefully a better way of checking wether there is already feedback stored for a user.